$NetBSD: patch-gobject_meson.build,v 1.3 2019/06/16 15:28:12 leot Exp $

Leave glib-genmarshal and glib-mkenums to devel/glib2-tools.
 
--- gobject/meson.build.orig	2019-06-10 17:47:20.000000000 +0000
+++ gobject/meson.build
@@ -75,7 +75,7 @@ foreach tool: python_tools
     input : tool + '.in',
     output : tool,
     configuration : python_tools_conf,
-    install : true,
+    install : false,
     install_dir : glib_bindir,
   )
 
